### ISSUE TITLE
Making the navbar responsive

### ISSUE THREAD

**Issue by c4dr-me** [Bitbox-Connect/Bitbox#76]
The navbar is not responsive. I would like to fix that
![image](https://github.com/user-attachments/assets/dc1851ef-732e-413e-8561-9bb20df65a5a)

**Comment by Anuj3553** [comment]
Hyy @c4dr-me, I'm assigning this issue to you.

**Comment by Anuj3553** [comment]
Hyy @c4dr-me, Can you give star to this repo?

**Comment by c4dr-me** [comment]
> Hyy @c4dr-me, Can you give star to this repo?

thank you! i already did
Will start working on it

**Comment by c4dr-me** [comment]
![image](https://github.com/user-attachments/assets/3c2c1799-6a12-4da4-aa04-9639dbdfc114)
is the deployed version and production version not same? the navbar doesn't seem to show anything on localhost

**Comment by Anuj3553** [comment]
> ![image](https://private-user-images.githubusercontent.com/142721314/375819448-3c2c1799-6a12-4da4-aa04-9639dbdfc114.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2NjUwNTAsIm5iZiI6MTcyODY2NDc1MCwicGF0aCI6Ii8xNDI3MjEzMTQvMzc1ODE5NDQ4LTNjMmMxNzk5LTZhMTItNGRhNC1hYTA0LTk2MzlkYmRmYzExNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMVQxNjM5MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hY2VhZDFlZDcwMDBmNGZhMGJlYjliNzExNGExM2U0N2RlZmY3ZmU5N2EyOTQ5NzVhY2JjNTRhNWYzOTVlZTkyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.kfmKskkyLETkNygoJYUtlt-z-8MB2UxzkkBYLxBOWvQ) is the deployed version and production version not same? the navbar doesn't seem to show anything on localhost

Yes, they are different. I don't want to make effect on my production website.

**Comment by Anuj3553** [comment]
I have updated your level because you solve the bugs in the navbar. Well done @c4dr-me.

**Comment by c4dr-me** [comment]
thank you very much! @Anuj3553

**Comment by c4dr-me** [comment]
there a lot of bugs and css changes that were different in production and deployed version. I can make more issues regarding them?

**Comment by Anuj3553** [comment]
Yes, The reason is that i don't want to manipulate my website and i will update my production website soon as possible. Yes you can raise your new issue.
